cdc/puller/sorter/metrics.go

```diff
@@ -18,67 +18,75 @@ import (
 )
 
 var (
+	sorterConsumeCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "ticdc",
+		Subsystem: "sorter",
+		Name:      "consume_count",
+		Help:      "the number of events consumed by the sorter",
+	}, []string{"capture", "changefeed", "table", "type"})
+
 	sorterEventCount = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "ticdc",
 		Subsystem: "sorter",
-		Name:      "sorter_event_count",
```

Comment:
Change metrics name may lead to no data point in grafana dashboard. Please make sure it's backward compatible.

Reply:
This metrics was not published in the grafana dashboard.